### PR TITLE
Fix ADMM test

### PR DESCRIPTION
**Pull request by EnLAI111**

### CODE REVIEW

**Comment by agramfort** [comment]
Why do you need this @EnLAI111 ?

File: solvers/ADMM.py [hunk 1]
```diff
@@ -1,6 +1,10 @@
 from benchopt import BaseSolver
 from benchopt.stopping_criterion import SufficientDescentCriterion
 from benchopt import safe_import_context
+from benchopt.utils.safe_import import set_benchmark
+
+# this means this test has to be run from the root
+set_benchmark('.')
```
Comment (tomMoral) [inline solvers/ADMM.py:7]: This is necessary to make `import_ctx.import_from` work

**Comment by EnLAI111** [comment]
> Why do you need this @EnLAI111 ?

I wrote a test of a function and its jacobian matrix in ```ADMM```, and they are based on other functions imported by ```import_ctx.import_from()``` form other files. Without this, the test can't work.

**Comment by tomMoral** [comment]
Actually, I read too fast.
You should not put this in the `ADMM.py` file but in the test file.

**Pull request merged**